bower.json

```diff
@@ -16,17 +16,21 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-nonempty": "^4.0.0",
+    "purescript-nonempty": "matthewleon/purescript-nonempty#Foldable1",
```

Comment:
Does this dependency need changing at all? Seems like we could remove it even, after this (although, that would make it breaking).

Reply:
Ah, I guess the tests might need it?

Reply:
Yes, though perhaps there is a way to change that. I'll look into it. It would be a pity to make the change breaking just for tests.

Reply:
Actually the dep is also still used in `Control.Monad.Gen.Common` for `genNonEmpty`. Now, given that `NonEmpty` is made an instance of `Foldable1` with https://github.com/purescript/purescript-nonempty/pull/27 it maybe makes sense to kill that function. What do you think?

Reply:
It's not a breaking change to update bounds, unless the bound change is also breaking, so I think it's fine.

Maybe we can move `genNonEmpty` to `NonEmpty` when we next break it - we don't want to delete it, the changes here don't give us a way of generating `Foldable1` :)

Reply:
I merged/released `-nonempty` with the `Foldable1` instance now too, so we can use the real dependency here too!